otter/test/convergence/test_service.py

```diff
@@ -857,7 +858,7 @@ def setUp(self):
         self.now = datetime(1970, 1, 1)
         self.waiting = Reference(pmap())
 
-    def get_seq(self, with_cache=True):
+    def get_seq(self, with_cache=True, upd_status_nogroup_err=False):
```

Comment:
Oh it needs more than a docstring. Its one of those messy tests that I would like to rewrite. However, I'll add a docstring to explain.